testing/viewnior/APKBUILD

```diff
@@ -1,33 +1,24 @@
 # Contributor: August Klein <amatcoder@gmail.com>
 # Maintainer: August Klein <amatcoder@gmail.com>
 pkgname=viewnior
-pkgver=1.5
+pkgver=1.6
 pkgrel=0
 pkgdesc="A fast and simple image viewer"
 url="http://siyanpanayotov.com/project/viewnior"
 arch="all"
 license="GPL3"
 depends=""
-depends_dev="gtk+2.0-dev exiv2-dev"
-makedepends="$depends_dev autoconf automake gnome-common libtool"
+depends_dev=""
+makedepends="autoconf automake gnome-common gtk+2.0-dev exiv2-dev libtool"
 install=""
 subpackages="$pkgname-lang $pkgname-doc"
 source="https://github.com/xsisqox/Viewnior/archive/$pkgname-$pkgver.tar.gz"
 
-_builddir="$srcdir"/Viewnior-$pkgname-$pkgver
-prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-	./autogen.sh || return 1
-}
+builddir="$srcdir"/Viewnior-$pkgname-$pkgver
 
 build() {
-	cd "$_builddir"
+	cd "$builddir"
+	./autogen.sh || return 1
```

Comment:
I've missed this line before; it should stay in the `prepare()` function. You can call `default_prepare` inside `prepare` to keep the default behaviour, i.e. apply patches.

```sh
prepare() {
	default_prepare || return 1

	cd "$builddir"
	./autogen.sh
} 
```